doc/content/gateways/lora-basics-station/implementation-details.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Implementation Details"
```

Comment:
Since this is very technical and probably not applicable for normal users of LBS, let's move it to the reference section and title it "LoRa Basics Station Implementation Note".